docs/management.md

```diff
@@ -293,42 +299,63 @@ rabbitmqctl set_user_tags full_access administrator
 
 ## Authenticating with OAuth 2 {#oauth2-authentication}
 
-RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
-not automatically redirect users to authenticate
-against the OAuth 2 server, this must be configured separately. Currently,
-**Authorization code flow with PKCE** is tested with the following Authorization servers:
+RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
+not automatically redirect users to authenticate against the OAuth 2 server, this must be configured separately. Currently, **Authorization code flow with PKCE** is tested with the following Authorization servers:
 
```

Comment:
This is my take on the information above:

**RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2) to authenticate client applications and management UI users.** So by default RabbitMQ is not configured to use JWT-encoded OAuth 2.0 access tokens, is that right??

Do we need to tell them what the default configuration is then say "you can configure RabbitMQ to use JWT-encoded OAuth 2.0 access tokens to authenticate client applications and management UI users" The benefits  of doing this are.....Then go onto say "When you are configuring RabbitMQ to use JWT-encoded OAuth 2.0 access tokens, remember that the management UI does
not automatically redirect users to authenticate against the OAuth 2 server. You must configure this separately. Currently, **Authorization code flow with PKCE** is tested with the following Authorization servers which you can use: